tpu-client/src/tpu_client.rs

```diff
@@ -187,6 +267,35 @@ where
     }
 }
 
+impl<P, M, C> AsyncClient for TpuClient<P, M, C>
+where
+    P: ConnectionPool<NewConnectionConfig = C>,
+    M: ConnectionManager<ConnectionPool = P, NewConnectionConfig = C>,
+    C: NewConnectionConfig,
+{
+    fn async_send_versioned_transaction(
```

Comment:
Looks like these 2 functions are not used anywhere in this PR. Could you comment on why they are included?

Reply:
ya these methods are required/used by `client.async_transfer()`. See: https://github.com/gregcusack/solana/blob/57cede46cdd58e5f67e38bcef5783c670f12171d/local-cluster/tests/local_cluster.rs#L2925 and https://github.com/gregcusack/solana/blob/57cede46cdd58e5f67e38bcef5783c670f12171d/local-cluster/tests/local_cluster.rs#L2936